### PR TITLE
Update us_pluto.m3u

### PR DESCRIPTION
**Pull request by BellezaEmporium**
2024-06-22

### CODE REVIEW

File: streams/us_pluto.m3u [hunk 1]
```diff
@@ -1,1145 +1,861 @@
...
-#EXTINF:-1 tvg-id="TheNewDetectives.us",Pluto TV The New Detectives (720p) [Not 24/7]
-https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5e393d5c696b3b0009775c8b/master.m3u8?advertisingId=channel&appName=rokuchannel&appVersion=1.0&bmodel=bm1&channel_id=channel&content=channel&content_rating=ROKU_ADS_CONTENT_RATING&content_type=livefeed&coppa=false&deviceDNT=1&deviceId=channel&deviceMake=rokuChannel&deviceModel=web&deviceType=rokuChannel&deviceVersion=1.0&embedPartner=rokuChannel&genre=ROKU_ADS_CONTENT_GENRE&is_lat=1&platform=web&rdid=channel&studio_id=viacom&tags=ROKU_CONTENT_TAGS
-#EXTINF:-1 tvg-id="TheNewDetectives.us",Pluto TV The New Detectives (480p)
-https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5ea71d48af1d0b0007d837f4/master.m3u8?advertisingId=&appName=web&appVersion=5.14.0-0f5ca04c21649b8c8aad4e56266a23b96d73b83a&app_name=web&clientDeviceType=0&clientID=6fbead95-26b1-415d-998f-1bdef62d10be&clientModelNumber=na&deviceDNT=false&deviceId=6fbead95-26b1-415d-998f-1bdef62d10be&device
```
Comment (BellezaEmporium) [inline streams/us_pluto.m3u:370]:
```suggestion
#EXTINF:-1 tvg-id="KevinHartsLOLNetwork.us",Kevin Hart’s LOL! Network
```

Comment (BellezaEmporium) [inline streams/us_pluto.m3u:406]:
```suggestion
#EXTINF:-1 tvg-id="Miobsesionfavorita.us",Mi obsesión favorita
```

Comment (BellezaEmporium) [inline streams/us_pluto.m3u:612]:
```suggestion
#EXTINF:-1 tvg-id="pocketwatch.us",pocket.watch
```

Comment (BellezaEmporium) [inline streams/us_pluto.m3u:644]:
```suggestion
#EXTINF:-1 tvg-id="SeriesconN.us",Series con Ñ
```

Comment (BellezaEmporium) [inline streams/us_pluto.m3u:660]:
```suggestion
#EXTINF:-1 tvg-id="Spikeextremo.us",Spike ¡extremo!
```

Comment (BellezaEmporium) [inline streams/us_pluto.m3u:696]:
```suggestion
#EXTINF:-1 tvg-id="Telemundotelenovelasclasicas.us",Telemundo telenovelas clásicas
```

Comment (BellezaEmporium) [inline streams/us_pluto.m3u:802]:
```suggestion
#EXTINF:-1 tvg-id="VevoIconosLatinos.us",Vevo Íconos Latinos
```

Comment (BellezaEmporium) [inline streams/us_pluto.m3u:812]:
```suggestion
#EXTINF:-1 tvg-id="VevoReggaetonTrap.us",Vevo Reggaeton & Trap
```

**Comment by Dum4G** [comment]
Geo-blocked for Russia

**Review by Dum4G — DISMISSED** [review]

**Review by Dum4G — APPROVED** [review]

**Review by Sphinxroot — APPROVED** [review]

**Pull request merged**